tests/test_mocker.py

```diff
@@ -243,6 +248,16 @@ def test_copy(self):
         self.assertEqual(copy_of_mocker._kw, mocker._kw)
         self.assertEqual(copy_of_mocker.real_http, mocker.real_http)
 
+    def test_reset_mock_calls_adapter_reset_mock(self):
+        with requests_mock.Mocker() as mocker:
+            self.assertMockStarted()
+
+            # Verify mocker.reset_mock calls reset_mock on adapter
+            with patch.object(mocker._adapter, 'reset_mock') as m_adapter_reset:
+                mocker.reset_mock()
+                # Assert adapter reset calls reset on matcher
+                m_adapter_reset.assert_called_once()
+
```

Comment:
I'd prefer this test was basically a clone of test_adapter. IMO that mocker calls adapter for this is an implementation detail, what we care about is that mocker.reset_mock performs as you'd expect.

Reply:
This may just be a style thing.  I am used to doing unit tests.  I will take your guidance.